nimble/host/syscfg.yml

```diff
@@ -101,6 +101,10 @@ syscfg.defs:
         description: 'Security manager secure connections (4.2).'
         value: 0
 
+    BLE_SM_USE_TINYCRYPT:
```

Comment:
I'd make this choice option

BLE_SM_CRYPTO:
    value: tinycrypt
    choices:
        - tinycrypt
        - mbedtls

Although mbedtls seems to be bigger than tinycrypt it may stil be preferred if application is already using it for other reasons

Reply:
@kasjer @sjanc what plan we have about this?